Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ ly_add_target(
             Gem::LmbrCentral.API
 )
 
-target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs tf2_ros ackermann_msgs gazebo_msgs)
+target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs tf2_ros ackermann_msgs gazebo_msgs vision_msgs)
```

Comment:
![image](https://github.com/o3de/o3de-extras/assets/12022617/c9c41ac9-e0cb-44e4-bb42-f01723b241fc)
Current documents do not have any documentation on this, this could be made as a separate pr

Reply:
Added change to documentation https://github.com/o3de/o3de.org/pull/2569